test/controllers/hello_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class HelloControllerTest < ActionDispatch::IntegrationTest
+  # test "the truth" do
+  #   assert true
+  # end
+end
```

Comment:
Did this come standard or did you add this?

Reply:
It was automatically created when running the generator command `rails g react:component HelloWorld greeting:string`